l10n_it_declaration_of_intent/models/account_move.py

```diff
@@ -220,6 +220,8 @@ def get_declarations(self):
                 partner_id=self.partner_id.id,
                 date=self.invoice_date,
             )
+        ids = [dec.id for dec in sorted(declarations, key=lambda d: d.date_start)]
+        declarations = declaration_model.browse(ids)
```

Comment:
Perché non fare direttamente:

declarations = sorted(declarations, key=lambda d: d.date_start)

Reply:
La funzione ritorna un recordset, la `sorted` no:

> Recordsets are iterable so the usual Python tools are available for transformation (map(), sorted(), ifilter(), …) however these return either a list or an iterator, removing the ability to call methods on their result, or to use set operations.

e non mi andava di cambiare la signature del metodo.

Ovviamente andando a vedere il metodo viene chiamato - da codice attuale - solo una volta e poi il risultato viene usato solo come iteratore in vari `for` per cui - praticamente - funzionerebbe uguale.

Reply:
Sarebbe equivalente di questo no?
```suggestion
        declarations = declarations.sorted(key="date_start")
```